scripts/docker/run_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Given that this script is not exclusive to docker images, I would put it in /scripts, to be used by local setups alike

Reply:
Done 2dfdbc1e1bb3c8983b5f873aff70104796b7e591